packages/@o3r/core/package.json

```diff
@@ -68,6 +68,7 @@
   },
   "devDependencies": {
     "@ama-sdk/core": "workspace:^",
+    "@ama-sdk/schematics": "workspace:^",
```

Comment:
Is it enough a devDep ?
Because you are calling _externalSchematic_ on this package ? Is this one doing an install too ?

Reply:
good point, an optional peerDep can make sense at least to indicate peerDep version